### PR TITLE
dist/deb: support make deb packages

### PR DESCRIPTION
**Pull request by houhuiting**
Signed-off-by: Huiting Hou <houhuiting@pku.stu.edu.cn>

### CODE REVIEW

File: dist/deb/build.sh [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
-RELEASE_TARBALL_URL=https://github.com/alibaba/inclavare-containers/archive/v$VERSION.tar.gz
 TARBALL_NAME=$PACKAGE\_$VERSION.orig.tar.gz
 DEB_BUILD_FOLDER=$DEBBUILD_DIR/$PACKAGE-$VERSION
 
 # create and rename the tarball
 mkdir -p $DEBBUILD_DIR
 if [ ! -f "$RELEASE_TARBALL" ]; then
-  wget -P $DEBBUILD_DIR $RELEASE_TARBALL_URL
+  cp -r $PROJECT_DIR $DEB_BUILD_FOLDER
 fi
```
Comment (haosanzi) [inline dist/deb/build.sh:15]: please fix the tab format

File: dist/deb/build.sh [hunk 2]
```diff
@@ -30,9 +25,34 @@ if [ ! -d "$SGX_SDK" ]; then
...
+        rm -rf $DEB_BUILD_FOLDER/src
+        cp -r $PROJECT_DIR/src $DEB_BUILD_FOLDER/src
+        cp -rf  $SCRIPT_DIR/debian $DEB_BUILD_FOLDER
+        cd $DEB_BUILD_FOLDER
+	sed 's/Package: rats-tls/Package: rats-tls-'$BUILD_MODE'/g' debian/control.in > debian/control 
+        sed 's/cmake -DBUILD_SAMPLES=on -H. -Bbuild/cmake -DRATS_TLS_BUILD_MODE="'$BUILD_MODE'" -DBUILD_SAMPLES=on -H. -Bbuild/g' debian/rules.in > debian/rules
+        sed -i 's/rats-tls-host/rats-tls-'$BUILD_MODE'/g' debian/rules
+        if [ $BUILD_MODE == "sgx" ]; then
+                sed -i "/override_dh_strip/adh_strip --exclude=rats-tls-server --exclude=rats-tls-client --exclude=sgx_stub_enclave.signed.so --exclude=librats_tls.a --exclude=librats_tls_u.a --exclude=librtls_edl_t.a --exclude=libtls_wrapper*.a --exclude=libcrypto_wrapper*.a --exclude=libattester*.a --exclude=libverifier*.a" debian/rules
+	        sed -i 's/dh_strip --exclude/\tdh_strip --exclude/g' debian/rules
```
Comment (haosanzi) [inline dist/deb/build.sh:46]: useless command?

File: dist/deb/debian/rules.in [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/make -f
-BUILD_ROOT := $(CURDIR)/debian/rats-tls
+BUILD_ROOT := $(CURDIR)/debian/rats-tls-host
```
Comment (YangLiang3) [inline dist/deb/debian/rules.in:2]: why append "host"  as suffix?

Comment (houhuiting) [reply dist/deb/debian/rules.in:2]: We no longer build the package of rats-tls, but build the package of rats-tls in host、tdx、occlum and sgx mode, so it is named here as rats-tls-host

**Review by haosanzi — APPROVED** [review]

**Review by YangLiang3 — APPROVED** [review]

**Pull request merged**